packages/es-dev-server/test/middleware/compatibility-transform.test.js

```diff
@@ -117,7 +117,7 @@ describe('compatibility transform middleware', () => {
     it('does not do any transformation for standard features', async () => {
       await expectCompatibilityTransform(userAgents['Chrome 78'], {
         objectSpread: false,
-        templateLiteral: true,
+        templateLiteral: false,
```

Comment:
@LarsDenBakker the three changes in this file are the ones I'd be the most obliged that you confirmed. I think this falls into the realm of "there are e2e tests for that" as I understood your previous comment, but wanted to double-check as everything else I changes was just package numbers and API properties.

Reply:
This is a correct and expected cange!